damus/Util/Markdown.swift

```diff
@@ -20,7 +20,9 @@ public struct Markdown {
         let md_opts: AttributedString.MarkdownParsingOptions =
             .init(interpretedSyntax: .inlineOnlyPreservingWhitespace)
 
-        if let txt = try? AttributedString(markdown: content, options: md_opts) {
+        // TODO: escape unintentional markdown
+        var escaped = content.replacingOccurrences(of: "\\_", with: "\\\\\\_")
```

Comment:
escaping all backslashes is unnecessary (see tests), so we only need to escape the ones where backslash does escaping for a valid markdown char, like `_`.

Reply:
Yeah, I'm not too happy with this hack, but when note content can be sometime markdown, sometimes regular text, we need something to handle to corner cases.